### ISSUE TITLE
Game Overlay Won't Open

### ISSUE THREAD

**Issue by SilvertongueRED** [kevinlekiller/reshade-steam-proton#34]
So, I installed this the quick way and it looks like it individually went into each of my game folders. However, the one I'm currently trying to ReShade is Loop Hero and despite editing the Reshade.ini "KeyOverlay" value to be everything but the 36 (home) button as I don't have one on my keyboard (like "KeyOverlay=220,0,0,0" where 220 is \) the overlay won't open and I can't reshade anything despite making Steam force the compatibility tool to use the latest Proton to open Loop Hero. It should be noted that I'm on a Steam Deck in desktop mode currently trying all of this. Any help much appreciated as I don't know what I'm doing!

**Comment by alexispurslane** [comment]
Yeah I ran this on a proton game and no key I've tried allows me to open the reshade overlay

**Comment by SilvertongueRED** [comment]
Oooh, ok, so: I actually got this to work! All I did was bind the home button key to a random controller button (use steam controller config). If that still doesn’t work, make sure you have the correct 32/64 bit done for the install along with the right Direct X version (try PC Gaming Wiki and search for your game then scroll/Ctrl+F for API to get right to the file architecture [32/64bit] and Direct X [dxgi, etc.] version). Ensure you’ve selected the correct install location for your game’s exe as well. Also make sure you copy those launch directions at end of install to the launch options section of your game in game details properties. Lemme know if these work if not I’ll try and think what else I did!

**Comment by alexispurslane** [comment]
OHHHH wait it isn't 64bit that might be the issue

**Comment by alexispurslane** [comment]
It works now!!! Script worked wonderfully

**Comment by SilvertongueRED** [comment]
Glad to hear! It’s an awesome script! I recommend defaulting your startup reshade to the fake hdr one!Roland DuBoisiCloudOn Nov 23, 2023, at 9:45 PM, Alexis Purslane ***@***.***> wrote:﻿
It works now!!! Script worked wonderfully

**Comment by LaserMoai** [comment]
@Slvrtnge 
Hello! Nothing here worked for me so far.
